zhaquirks/kof/kof_mr101z.py

```diff
@@ -56,7 +56,7 @@ async def command(self, command, *args, expect_reply=None, **kwargs):
         else:
             cmd_expect_reply = expect_reply
 
-        rsp = await super(NoReplyMixin, self).command(
+        rsp = await super().command(
```

Comment:
@puddly will this break anything??

Reply:
Don't think so. This code is explicitly unit tested I think.